scripts/apps/workspace/content/directives/ContentProfileSchemaEditor.ts

```diff
@@ -93,6 +91,7 @@ const EDITOR3_RICH_FORMATTING_OPTIONS = [
     'table',
     'formatting marks',
     'remove format',
+    'remove all format',
```

Comment:
It should probably be called "remove formatting", not "remove format".

Reply:
I thought I read it this way on the requirements but I can't see it anymore so I'll change it cause it makes more sense

Reply:
you mean "remove all formatting" or change the old one too?

Reply:
It'd be good to rename both since it's the same term.

Reply:
I think that would remove the option from current instances (as this is an id). Also, I think I'm gonna put all these into a type so we can keep track of them in other files

Reply:
You can keep the ID and only change naming in function/variable names.